addons/im_livechat/demo/im_livechat_channel/im_livechat_support_bot_session_1.xml

```diff
@@ -86,5 +86,86 @@
             <field name="user_script_answer_id" ref="chatbot_script_troubleshooting_answer_chrome_demo"/>
             <field name="user_raw_script_answer_id" ref="chatbot_script_troubleshooting_answer_chrome_demo"/>
         </record>
+        <record id="support_bot_session_1_message_5" model="mail.message">
+            <field name="model">discuss.channel</field>
+            <field name="res_id" ref="support_bot_session_1_demo"/>
+            <field name="author_id" ref="support_bot_operator_partner_demo"/>
+            <field name="body">What module are you interested in?</field>
+            <field name="message_type">comment</field>
+            <field name="subtype_id" ref="mail.mt_comment"/>
+            <field name="date" eval="DateTime.today() + relativedelta(months=-1, minutes=5)"/>
```

Comment:
every message should have `create_date` as well as this field is used to compute statistics such as `duration`/`time_to_answer`.